README.adoc

```diff
@@ -1,4 +1,4 @@
-= Spring Cloud Deployer for Cloud Foundry image:https://build.spring.io/plugins/servlet/wittified/build-status/SCD-DCF[Build Status, link=https://build.spring.io/browse/SCD-DCF]
```

Comment:
I question the value of having build status in the README even when the link is accessible.